### ISSUE TITLE
Add Windows Support

### ISSUE THREAD

**Issue by SrGato001** [RTUnit/Jellyfin-Transcodes-cleanup#7]
Hey
It would be cool to have support for Jellyfin windows (without docker) :D

**Comment by RTUnit** [comment]
Yes.. that would be great for those using Jellyfin, Emby or Plex on Windows. That would be possible either by:

1) developing a script based on a number of command-line tools (like this one for pausing/resuming Windows processes: https://www.codeproject.com/Articles/2964/Win32-process-suspend-resume-tool) or

2) developing a all-in-one Windows program.

Anyway, that would be a separate project, not a continuation of this repo :-)

I will leave this issue open just in case if someone takes initiative and creates a new project.
